woocommerce/woocommerce
# [4.0] Remove wc-admin install nudge.

### Changes proposed in this Pull Request:

This branch removes the admin alert that promotes the WooCommerce Admin feature plugin.

Closes #25488 .

### How to test the changes in this Pull Request:

1. Install this branch on a site that does not have woocommerce admin installed
2. Verify no admin alert is displayed to install wc-admin

### Changelog entry

> Remove WooCommerce Admin install alert
